resources/dicts/events/death/general/general.json

```diff
@@ -763,6 +763,39 @@
         "history_text": {
             "reg_death": "This cat died from old age life.",
             "lead_death": "lost all remaining lives from old age."
-        }
+        },
+            {
+    "camp": "any",
+    "tags": [
+        "other_cat",
+        "Newleaf",
+        "Greenleaf",
+        "Leaf-fall",
+        "Leaf-bare"
+    ],
+    "death_text": "In the middle of a conversation, r_c suddenly points out the circling shadow, slowly getting closer to the ground. m_c freezes, looking up at the predator, unable to move from sudden shock. r_c tries to shove them out of the way, but the large bird has already fastened its grip around m_c, and it's too late.",
```

Comment:
I recommend you make this a mountains specific death, and add an injury to r_c 

I also think the event is too long, and some of these words aren't pulling their weight for you. Recommend "In the middle of a conversation, r_c suddenly points out the circling shadow, slowly getting closer to the ground." --> "r_c, guarding camp, yells a warning as they spot a circling shadow." (this would require you to specify that r_c can only be a warrior)

Reply:
👍